MixedRemoteViewCompositor/Source/Shared/Network/Connection.cpp

```diff
@@ -434,6 +434,121 @@ HRESULT ConnectionImpl::SendBundleAsync(
     return spWriteAction.CopyTo(sendAction);
 }
 
+_Use_decl_annotations_
+HRESULT ConnectionImpl::SendBundleAsyncSequenced(

```

Comment:
can the single buffer/bundle be refactored to use SendBundleAsync function so there is only one function that is possibly writing to the output stream of the socket

Reply:
Done. Removed the SendBundleAsyncSequenced function and refactor its code into the original function SendBundleAsync. Please check the latest commit.